docs/README.md

```diff
@@ -1905,7 +1905,54 @@ See more details in [monitoring docs](#monitoring).
 
 See [this article](https://valyala.medium.com/how-victoriametrics-makes-instant-snapshots-for-multi-terabyte-time-series-data-e1f3fb0e0282) for more details.
 
-See also [how to work with snapshots](#how-to-work-with-snapshots).
+See also [how to work with snapshots](#how-to-work-with-snapshots) and [IndexDB](#indexdb).
+
+## IndexDB
+
+VictoriaMetrics identifies
+[time series](https://docs.victoriametrics.com/keyconcepts/#time-series) by
+`TSID` (time series ID) and stores
+[raw samples](https://docs.victoriametrics.com/keyconcepts/#raw-samples) sorted
+by TSID (see [Storage](#storage)). Thus, the TSID is a primary index and could
+be used for searching and retrieving raw samples. However, the TSID is never
+exposed to the clients, i.e. it is for internal use only.
+
+Instead, VictoriaMetrics maintains an **inverted index** that enables searching
+the raw samples by metric name, label name, and label value by mapping these
+values to the corresponding TSIDs.
+
+VictoriaMetrics uses two types of inverted indexes:
+
+-   Global index. This was the original type of index used by VictoriaMetrics.
+    Searches using this index is performed across the entire retention period.
+-   Per-day index. This index was introduced later as an optimization for
+    handling.
+    [high churn rate](https://docs.victoriametrics.com/faq/#what-is-high-churn-rate)
+    (frequent creation and deletion of time series). The per-day index stores
+    mappings similar to ones in global index but also includes the date in each
+    mapping. This speeds up data retrieval for queries within a shorter time
+    range (which is often just the last day).
+
+When the search query is executed, VictoriaMetrics decides which index to use
+based on the time range of the query:
+
+-   Per-day index is used if the search time range is 40 days or less.
+-   Global index is used for search queries with a time range greater than 40
+    days.
+
+Mappings are added to the indexes during the data ingestion:
+
+-   In global index each mapping is created only once per retention period.
+-   In the per-day index each mapping will be created for each unique date that
+    has been seen in the samples for the corresponding time series.
+
+NOTE: The mappings are not created atomically and duplicates are possible when
```

Comment:
I don't see value in this sentence. It should be either removed or linked to some behavior that user can experience or alter. In other words, it has no practical knowledge for a user.

Reply:
Removed.